pyproject.toml

```diff
@@ -18,19 +21,19 @@ playwright = "^1.35.0"
 readability-lxml = "^0.8.1"
 bleach = "^5.0.1"
 python-json-logger = "^2.0.4"
-cloudpathlib = {extras = ["s3"], version = "^0.10.0"}
+cloudpathlib = { extras = ["s3"], version = "^0.10.0" }
 PyMuPDF = "^1.20.2"
 google-cloud-translate = "^3.8.2"
 psutil = "^5.9.2"
 multiprocessing-logging = "^0.3.3"
 json-logging = "^1.3.0"
 tenacity = "^8.2.1"
-pillow = "<10"  # to fix import of Image.LINEAR by dependency
+pillow = "<10"                                                                                            # to fix import of Image.LINEAR by dependency
```

Comment:
Do we need to remove these spaces?

Reply:
🤷 autoformatter really wants those spaces in there for some reason!

Reply:
no effect on the code, so i'm going to leave them for now - hope that's okay!